File: 프로그래머스/0/181870. ad 제거하기/ad 제거하기.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(vector<string> strArr) 
{
    vector<string> v;
    for(string str : strArr)
    {
        if (str.find("ad") != string::npos)
        {
            continue;
        }
        v.push_back(str);
    }
    return v;
}

File: 프로그래머스/0/181870. ad 제거하기/ad 제거하기.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public string[] solution(string[] strArr) 
    {
        List<string> strList = new List<string>();
        
        for(int i=0; i<strArr.Length; i++) {
            if(strArr[i].Contains("ad")) {
                continue;
            }
            strList.Add(strArr[i]);
        }
        return strList.ToArray();
    }
}